plugins/providers/gcloudiam/config.go

```diff
@@ -147,8 +147,8 @@ func (c *Config) validateResourceConfig(resource *domain.ResourceConfig) error {
 	}
 
 	if resource.Roles == nil || len(resource.Roles) == 0 {
-		return nil
+		return ErrRolesShouldNotBeEmpty
 	}
 
-	return ErrShouldHaveEmptyRoles
```

Comment:
Can you also validate permissions like how it's done in `bigquery`? https://github.com/odpf/guardian/blob/main/plugins/providers/bigquery/config.go#L169

Reply:
Done